plone/app/contenttypes/testing.py

```diff
@@ -72,6 +81,8 @@ class PloneAppContenttypesMigration(PloneSandboxLayer):
     defaultBases = (PLONE_FIXTURE,)
 
     def setUpZope(self, app, configurationContext):
```

Comment:
I would use the @unittest.skipUnless(TEST_MIGRATION, "Test only when Archetype is relevant") decorator, but this is not a blocker for merging!

Reply:
Wow! You are always a source for new knowledge :)
Would you mind adding a changelog line, otherwise I will do that myself.

Reply:
Well, I learned it myself the other day by looking at the source code when it did not work. :) Changelog added.